packages/matter-node-shell.js/src/util/String.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * @license
+ * Copyright 2022-2023 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export function firstLetterToLowerCase(str: string) {
```

Comment:
I use "camelize" for this, kind of overkill but shorter to type :)

Reply:
Changed